mkdocs.yml

```diff
@@ -15,7 +15,9 @@ theme:
   name: material
   features:
     - content.tooltips
-    - navigation.indexes
+        - navigation.expand    
```

Comment:
```suggestion 
```
It turns out navigation.expand doesn't do what I want. It will expand all collpasable left bar sections, which will just overflow user's screens. Let's just leave this to navigation.path